ReactCommon/turbomodule/core/platform/ios/RCTTurboModuleManager.mm

```diff
@@ -67,19 +68,47 @@ - (instancetype)initWithRuntime:(jsi::Runtime *)runtime
 
       __strong __typeof(self) strongSelf = weakSelf;
 
+      auto moduleName = name.c_str();
+      bool didFindModuleInCacheBeforeCreation = strongSelf->_turboModuleCache.find(moduleName) != strongSelf->_turboModuleCache.end();
+      if (!didFindModuleInCacheBeforeCreation) {
+        [strongSelf->_bridge.performanceLogger markStartForTag:RCTPLTurboModuleSetup];
+      }
+
```

Comment:
nit: The variable name `didFindModuleInCacheBeforeCreation ` is a bit strange. What you're really checking here is: "Does this TurboModule exist?". So you could just use the `moduleIsInitialized` function defined further down in the file.

Reply:
Ahh, perfect - How did I miss that one out? 😅